examples/with-typescript/package.json

```diff
@@ -6,8 +6,10 @@
     "next": "latest"
   },
   "devDependencies": {
-    "@types/react": "^15.0.1",
+    "@types/react": "^15.0.21",
     "concurrently": "^3.1.0",
-    "typescript": "^2.1.5"
+    "typescript": "2.2.1",
+    "react":"latest",
+    "react-dom":"latest"
```

Comment:
latest react seems unsafe. i'd pick a version range just like above. how about `^15.4.0`?

also space after the colon.